effectiveOfficeBackend/src/main/kotlin/office/effective/features/user/routes/AuthRouting.kt

```
@@ -32,7 +34,12 @@ fun Route.authRoutingFun() {
         get("/callback") {
             try {
                 val principal: OAuthAccessTokenResponse.OAuth2? = call.principal()
```

Maybe you should throw an exception if principal is null?

---

Done